src/ConfigManager.cpp

```diff
@@ -342,6 +337,45 @@ void ConfigManager::createBaseWebServer() {
     server->onNotFound(std::bind(&ConfigManager::handleNotFound, this));
 }
 
+
+void ConfigManager::clearWifiSettings(bool reboot) {
+    char ssid[SSID_LENGTH];
+    char password[PASSWORD_LENGTH];
+    memset(ssid, NULL, SSID_LENGTH);
+    memset(password, NULL, PASSWORD_LENGTH);
+    storeWifiSettings(magicBytesEmpty, ssid, password);
+
+    if (reboot) {
+        ESP.restart();
+    }
+}
+
+void ConfigManager::storeWifiSettings(const char magicBytes[MAGIC_LENGTH], String ssid, String password) {
```

Comment:
Something in the latest commit killed properly setting WIFI conn. details -- fixing

Reply:
need to change this to a `String` for reasons beyond my control